packages/app/src/cli/prompts/config.ts

```diff
@@ -25,6 +31,21 @@ export async function selectConfigName(directory: string, defaultName = ''): Pro
   return configName
 }
 
+export async function selectConfigFile(directory: string): Promise<string | undefined> {
+  const files = (await promises.readdir(directory)).filter(
+    (file) => file.startsWith('shopify.app.') && file.endsWith('.toml'),
+  )
+
+  if (files.length < 2) return files[0]
+
+  return renderSelectPrompt({
+    message: 'Configuration file',
+    choices: files.map((file) => {
+      return {label: file, value: file}
+    }),
+  })
+}
```

Comment:
could we use https://github.com/Shopify/cli/blob/main/packages/cli-kit/src/public/node/result.ts so that the error case becomes clearer? happy to pair on this :)

Reply:
Oh I didn't know we had a type for Result here too, happy to use it 👍